tarpaulin.toml

```diff
@@ -22,4 +22,6 @@ exclude = [
     # repo automation (ci, codegen)
     "uniffi-bindgen",
     "xtask",
+    # temporary: tiny test suite, expensive to run
+    "sliding-sync-integration-test",
```

Comment:
ouch, we're going to lose all coverage for the notification client if we do that :/

Reply:
We were already excluding it, just using command line flags which seemed to not be respected some of the time.. If we want to run these tests, we need to set up the proxy in this CI job. Can be done in a follow-up PR.